gravity.sh

```diff
@@ -313,32 +313,16 @@ gravity_reload() {
 
 	# Reload hosts file
 	echo ":::"
-	echo -n "::: Refresh lists in dnsmasq..."
+	echo "::: Refresh lists in dnsmasq..."
 
 	#ensure /etc/dnsmasq.d/01-pihole.conf is pointing at the correct list!
 	#First escape forward slashes in the path:
 	adList=${adList//\//\\\/}
 	#Now replace the line in dnsmasq file
-	sed -i "s/^addn-hosts.*/addn-hosts=$adList/" /etc/dnsmasq.d/01-pihole.conf
+#	sed -i "s/^addn-hosts.*/addn-hosts=$adList/" /etc/dnsmasq.d/01-pihole.conf
 	find "$piholeDir" -type f -exec chmod 666 {} \;
 
-    dnsmasqPid=$(pidof dnsmasq)
-
-	if [[ ${dnsmasqPid} ]]; then
-	    # service already running - reload config
-	    if [ -x "$(command -v systemctl)" ]; then
-            systemctl restart dnsmasq
-        else
-            service dnsmasq restart
-        fi
-	else
-	    # service not running, start it up
-	    if [ -x "$(command -v systemctl)" ]; then
-            systemctl start dnsmasq
-        else
-            service dnsmasq start
-        fi
-	fi
+        pihole restartdnsmasq
```

Comment:
Can we name this `restartdns` instead. I don't want to start to get tied into one resolver. Make it a bit more general please.